packages/markedparser-extension/src/index.ts

```diff
@@ -22,20 +32,16 @@ const plugin: JupyterFrontEndPlugin<IMarkdownParser> = {
   id: '@jupyterlab/markedparser-extension:plugin',
   autoStart: true,
   provides: IMarkdownParser,
-  requires: [IEditorLanguageRegistry],
-  activate: (app: JupyterFrontEnd, languages: IEditorLanguageRegistry) => {
-    Private.initializeMarked(languages);
+  requires: [IEditorLanguageRegistry, IThemeManager],
```

Comment:
Maybe `IThemeManager` could be made optional?

Reply:
good call 385bdd7386